docs/interest-groups.html

```diff
@@ -43,14 +43,18 @@ <h3>Currently Joined</h3>
 		<script type="module">
 			import * as idb from 'https://cdn.jsdelivr.net/npm/idb-keyval@5/dist/esm/index.js';
 			import Fledge from './scripts/api/esm/index.js';
-			const fledge = new Fledge('https://magniteengineering.github.io/fledge.polyfill/iframe.html', true);
+			const fledge = new Fledge('http://localhost:8000/iframe.html', true);
 			const customStore = idb.createStore('fledge.v1', 'interest-groups');
 
 			document.getElementById("join-ig").onclick = async function joinGroup() {
 				await fledge.joinAdInterestGroup({
 					owner: document.getElementById('owner').value,
 					name: document.getElementById('name').value,
-					biddingLogicUrl: document.getElementById('url').value
+					biddingLogicUrl: document.getElementById('url').value,
+					trustedBiddingSignalsUrl: 'http://localhost:8000/get-values.json',
```

Comment:
should this be the `magniteengineering.github.io`?

Reply:
It will be, but I'm still testing.  Accidentally checked it in.